http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 03:39:33","{""user_id_"":""112"",""exception_"":""Attempt to invoke virtual method 'android.content.res.Resources$Theme android.content.Context.getTheme()' on a null object reference"",""class_name_"":""HomeScreen"",""method_name_"":""getMyLocation""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 03:39:43","{""user_id_"":""112"",""exception_"":""Attempt to invoke virtual method 'android.content.res.Resources$Theme android.content.Context.getTheme()' on a null object reference"",""class_name_"":""HomeScreen"",""method_name_"":""getMyLocation""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 03:39:43","{""user_id_"":""112"",""exception_"":""Attempt to invoke virtual method 'android.content.res.Resources$Theme android.content.Context.getTheme()' on a null object reference"",""class_name_"":""HomeScreen"",""method_name_"":""getMyLocation""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:25:08","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:25:09","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:25:10","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:25:10","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:25:11","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:25:11","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:30","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:31","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:32","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:32","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:33","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:33","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:30:34","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:48:34","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:48:46","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:49:57","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:50:10","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:50:23","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:50:52","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:51:10","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""verifybtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:51:19","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""verifybtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:51:27","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""verifybtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 06:51:31","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(493): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionVerifyotp()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('verifyotp', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/verifyotp', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""resndotpBtnClicked"",""class_name"":""OTPViewController"",""user_id"":""221""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 09:21:14","{""exception"":""(null)"",""method_name"":""savedStateForPendingTripFromServerWithWayPoints"",""class_name"":""Home"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 09:55:47","{""exception"":""(null)"",""method_name"":""serviceCallMethod"",""class_name"":""FavoritesVC"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 10:00:03","{""exception"":""{\n    code = 8;\n    file = \"\/var\/www\/html\/Tolls\/api\/components\/Myfunctions.php\";\n    line = 65;\n    message = \"Trying to get property of non-object\";\n    name = \"PHP Notice\";\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): api\\\\controllers\\\\TollController->actionCreatemonthlypass()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): ::call_user_func_array:{\/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php:55}()\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction()\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction()\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest()\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#7 {main}\"\n    );\n    type = \"yii\\\\base\\\\ErrorException\";\n}"",""method_name"":""transactionSuccessfulMethod"",""class_name"":""AddMonthlyPasses"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 10:01:43","{""exception"":""{\n    code = 8;\n    file = \"\/var\/www\/html\/Tolls\/api\/components\/Myfunctions.php\";\n    line = 65;\n    message = \"Trying to get property of non-object\";\n    name = \"PHP Notice\";\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): api\\\\controllers\\\\TollController->actionCreatemonthlypass()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): ::call_user_func_array:{\/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php:55}()\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction()\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction()\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest()\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#7 {main}\"\n    );\n    type = \"yii\\\\base\\\\ErrorException\";\n}"",""method_name"":""transactionSuccessfulMethod"",""class_name"":""AddMonthlyPasses"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 11:25:48","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 11:27:18","{""exception"":""{\n    Exception = \"View must not be nil.\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 11:32:49","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(186): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionSendresetpassword()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('sendresetpasswo...', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/sendresetp...', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 11:32:53","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(186): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionSendresetpassword()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('sendresetpasswo...', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/sendresetp...', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 11:36:43","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(186): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionSendresetpassword()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('sendresetpasswo...', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/sendresetp...', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 11:36:57","{""exception"":""{\n    code = 0;\n    file = \"\/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php\";\n    line = 265;\n    message = \"Connection could not be established with host *.prod.sin2.secureserver.net [php_network_getaddresses: getaddrinfo failed: Name or service not known #0]\";\n    name = Exception;\n    \"stack-trace\" =     (\n        \"#0 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/StreamBuffer.php(62): Swift_Transport_StreamBuffer->_establishSocketConnection()\",\n        \"#1 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Transport\/AbstractSmtpTransport.php(113): Swift_Transport_StreamBuffer->initialize(Array)\",\n        \"#2 \/var\/www\/html\/Tolls\/vendor\/swiftmailer\/swiftmailer\/lib\/classes\/Swift\/Mailer.php(79): Swift_Transport_AbstractSmtpTransport->start()\",\n        \"#3 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2-swiftmailer\/Mailer.php(140): Swift_Mailer->send(Object(Swift_Message))\",\n        \"#4 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMailer.php(260): yii\\\\swiftmailer\\\\Mailer->sendMessage(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#5 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/mail\/BaseMessage.php(48): yii\\\\mail\\\\BaseMailer->send(Object(yii\\\\swiftmailer\\\\Message))\",\n        \"#6 \/var\/www\/html\/Tolls\/api\/controllers\/UserController.php(186): yii\\\\mail\\\\BaseMessage->send()\",\n        \"#7 [internal function]: api\\\\controllers\\\\UserController->actionSendresetpassword()\",\n        \"#8 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/InlineAction.php(55): call_user_func_array(Array, Array)\",\n        \"#9 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Controller.php(151): yii\\\\base\\\\InlineAction->runWithParams(Array)\",\n        \"#10 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Module.php(455): yii\\\\base\\\\Controller->runAction('sendresetpasswo...', Array)\",\n        \"#11 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/web\/Application.php(83): yii\\\\base\\\\Module->runAction('user\/sendresetp...', Array)\",\n        \"#12 \/var\/www\/html\/Tolls\/vendor\/yiisoft\/yii2\/base\/Application.php(375): yii\\\\web\\\\Application->handleRequest(Object(common\\\\components\\\\Request))\",\n        \"#13 \/var\/www\/html\/Tolls\/api\/web\/index.php(18): yii\\\\base\\\\Application->run()\",\n        \"#14 {main}\"\n    );\n    type = \"Swift_TransportException\";\n}"",""method_name"":""btnResetCodeClicked"",""class_name"":""ForgotPasswordVC""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 12:41:22","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""212""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:28","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""216""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:31","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""212""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:31","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:37","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""216""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:37","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:37","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""212""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:45","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""216""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:46","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""212""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:46","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""216""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:47","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:47","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""212""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:47","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""21""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:54","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""212""}"
http://115.124.125.42/Tolls/api/2119/u/exceptionapi,"2016-05-11 13:28:55","{""exception"":""(null)"",""method_name"":""LocationManager:didEnterRegion"",""class_name"":""AppDelegate"",""user_id"":""21""}"
